content/content.js

```diff
@@ -13,27 +13,16 @@ const submitUrl = 'https://borang.skrin.xyz/submit';
   document.documentElement.append(div);
 })();
 
-let submitButton = document.querySelector(
-  // '.freebirdFormviewerViewNavigationSubmitButton'
-  // '.uArJ5e.UQuaGc.Y5sE8d.VkkpIf.NqnGTe'
-  '.uArJ5e.UQuaGc.Y5sE8d.VkkpIf'
-);
+const submitButton = document.querySelector('.uArJ5e.UQuaGc.Y5sE8d.VkkpIf.QvWxOd');
```

Comment:
IIRC, class `QvWxOd` is not consistent. It is dynamic. You can compare from my commented code. That's why I select `.uArJ5e.UQuaGc.Y5sE8d.VkkpIf` instead of `.uArJ5e.UQuaGc.Y5sE8d.VkkpIf.NqnGTe`.

I decided to comment out the code and not to remove the code for future reference

Reply:
Okay then, i will revert this and put comment on the loc.

Reply:
On second thought, all forms I checked, all submit buttons are having the `QvWxOd` class. But if you prefer to keep as it is, then I will proceed with the revert.

Reply:
Yup, it's also having `QvWxOd` on my side. However, From the commented code it used to be `NqnGTe` and it changes from time to time. I prefer to leave it as is